tests/console/hostname.pm

```diff
@@ -40,6 +41,13 @@ sub run {
         assert_script_run "ip addr show br0 | grep UP";
         save_screenshot;
     }
+
+    if (sle_version_at_least('15')) {
```

Comment:
That would match also on other unexpected versions, e.g. is true on TW. Better be explicit with `is_sle && sle_version_at_least('15')` but also check other versions. If all TW and SLE15 and Leap15.0 are showing the same behaviour - which they should - then it should rather be checking that it is not the opposite, e.g. check for `unless ((is_sle && !sle_version_at_least('15')) || (is_leap && !leap_version_at_least('15.0')))` but I am not very good with this logics so take it with a grain of salt :)